scripts/download-resources.mjs

```diff
@@ -80,53 +90,173 @@ export async function download(url, destPath, overwrite = false, access = fs.con
+  };
+
+  const osHomeDir = os.homedir();
+
+  if (osHomeDir && await tryAccess(osHomeDir)) {
+    return osHomeDir;
+  }
+  if (process.env.HOME && await tryAccess(process.env.HOME)) {
+    return process.env.HOME;
+  }
+  if (onWindows) {
+    if (process.env.USERPROFILE && await tryAccess(process.env.USERPROFILE)) {
+      return process.env.USERPROFILE;
+    }
+    if (process.env.HOMEDRIVE && process.env.HOMEPATH) {
+      const homePath = path.join(process.env.HOMEDRIVE, process.env.HOMEPATH);
+
+      if (await tryAccess(homePath)) {
+        return homePath;
+      }
+    }
   }
 
+  return null;
+}
+
+export default async function main() {
+  // Download Kubectl
+  const kubeVersion = (await (await fetch('https://dl.k8s.io/release/stable.txt')).text()).trim();
+  const kubectlURL = `https://dl.k8s.io/${ kubeVersion }/bin/${ kubePlatform }/amd64/${ exeName('kubectl') }`;
+  const kubectlPath = path.join(binDir, exeName('kubectl'));
+
+  fs.mkdirSync(binDir, { recursive: true });
+  // If kubectlPath is a symlink delete it before continuing
+  try {
+    const stat = await fs.promises.lstat(kubectlPath);
+
+    if (stat.isSymbolicLink()) {
+      await fs.promises.rm(kubectlPath);
+    }
+  } catch (_) {}
+  await download(kubectlURL, kubectlPath, false, fs.constants.X_OK);
+
+  // Download Helm. It is a tar.gz file that needs to be expanded and file moved.
+  const helmVersion = '3.6.0';
+  const helmURL = `https://get.helm.sh/helm-v${ helmVersion }-${ kubePlatform }-amd64.tar.gz`;
+
+  await downloadTarGZ(helmURL, 'helm', `${ kubePlatform }-amd64`);
+
   // Download Kim
   const kimVersion = '0.1.0-beta.2';
   const kimURL = `https://github.com/rancher/kim/releases/download/v${ kimVersion }/${ exeName(`kim-${ kubePlatform }-amd64`) }`;
   const kimPath = path.join(binDir, exeName('kim'));
 
   await download(kimURL, kimPath, false, fs.constants.X_OK);
+
+  const kuberlrVersion = '0.3.1';
+  const kuberlrBaseURL = `https://github.com/flavio/kuberlr/releases/download/v${ kuberlrVersion }/kuberlr_${ kuberlrVersion }_${ kubePlatform }_amd64`;
+  const kuberlrPlatformDir = `kuberlr_${ kuberlrVersion }_${ kubePlatform }_amd64`;
+  let kuberlrPath;
+
+  if (onWindows) {
+    kuberlrPath = await downloadZip(`${ kuberlrBaseURL }.zip`, 'kuberlr', kuberlrPlatformDir);
+  } else {
+    kuberlrPath = await downloadTarGZ(`${ kuberlrBaseURL }.tar.gz`, 'kuberlr', kuberlrPlatformDir);
+  }
+
+  // Desired:
+  // copy kubectl to ~/.kuberlr/PLATFORM-amd64/kubectlMAJ.MIN.PATCH
+  // .../resources/PLATFORM/bin: symlink kubectl pointing to kuberlr
+  if (kuberlrPath) {
+    const kuberlrDir = path.join(await findHome(), '.kuberlr', `${ kubePlatform }-amd64`);
+    const pathParts = path.parse(kubectlPath);
+    const newKubectlPath = path.join(kuberlrDir, `${ pathParts.name }${ kubeVersion.replace(/^v/, '') }${ pathParts.ext }`);
+
+    /*
+     * # The following code would do this in bash:
+     * if [-f $newKubectlPath] ; then
+     *   rm $kubectlPath
```

Comment:
This is not quite equivalent, since we also go into the `catch` block at line 246 if `rm` fails.

Reply:
Hmmm ... I wrote that out first, and then wrote the code. It's the case of a comment that is more likely to become obsolete than to help, so out it goes.